mage_ai/io/oracledb.py

```diff
@@ -24,7 +32,8 @@ def with_config(cls, config: BaseConfigLoader) -> 'OracleDB':
             user=config[ConfigKey.ORACLEDB_USER],
             password=config[ConfigKey.ORACLEDB_PASSWORD],
             host=config[ConfigKey.ORACLEDB_HOST],
-            port=config[ConfigKey.ORACLEDB_PORT]
+            port=config[ConfigKey.ORACLEDB_PORT],
+            service=config[ConfigKey.ORACLEDB_SERVICE]
```

Comment:
Did you test the case when the service is None? Just want to make sure both cases work.

Reply:
I have not tested yet, that's a todo

Reply:
Just tested it, by adding SERVICE_NAME as null ( or just not declaring it), code run as usual
![Screenshot from 2023-07-24 17-50-10](https://github.com/mage-ai/mage-ai/assets/14100959/67aea815-8173-4c52-8c9a-131856bc06be)
![Screenshot from 2023-07-24 17-52-43](https://github.com/mage-ai/mage-ai/assets/14100959/148a7415-39a9-47ca-a3ce-59442b657885)